### PR TITLE
new icon: web3js (original, plain)

### PR DESCRIPTION
**Pull request by shailendrakawadkar**
### Double check these details before you open a PR

- [x]  PR does not match another non-stale PR currently opened
- [x]  PR name matches the format new icon: Icon name (versions separated by comma). More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x]  PR's base is the develop branch.
- [x]  Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x]  SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x]  A new object is added in the devicon.json file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)
- [x] This PR closes https://github.com/devicons/devicon/issues/960

Link to prove your SVG is correct and up-to-date.
Logo PNG: 
image
Logo Preview: https://repository-images.githubusercontent.com/24655114/c71c5800-6a8c-11e9-9117-8ec357c9f69e
Github: https://github.com/ChainSafe/web3.js
Website: https://web3js.org/

### CODE REVIEW

**Comment by shailendrakawadkar** [comment]
@lunatic-fox @weh @quantik-git  please review this PR.

File: icons/web3js/web3-original.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
+<g id="surface1">
```
Comment (Snailedlt) [inline icons/web3js/web3-original.svg:3]: Looks like there's some white background left in the icon. Perhaps you could base your svg on the png used on their website?
image

File: devicon.json [hunk 2]
```diff
@@ -10455,6 +10455,25 @@
...
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#ffffff",
```
Comment (Snailedlt) [inline devicon.json:10474]: This seems to be the main color of the logo:
```suggestion
        "color": "#bcbec0",
```

File: devicon.json [hunk 3]
```diff
@@ -10455,6 +10455,25 @@
             }
         ]
     },
+    {
+        "name": "web3js",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:10460]:
```suggestion
        "altnames": [
            "web3.js"
        ],
```

File: icons/web3js/web3-plain.svg [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Snailedlt) [inline icons/web3js/web3-plain.svg:1]: plain icons should have only a single path. You can read more about our SVG standards on the wiki: https://github.com/devicons/devicon/wiki/SVG-Standards

My suggestion is to base the plain version on the original version, and just make it single-colored. Then you can unify the paths on this icon and call it web3-line.svg

**Review by Snailedlt — DISMISSED** [review]
Thanks for the contribution. Here are some things you need to change before it can be accepted

File: icons/web3js/web3-original.svg (hunk 1 above)
Comment (lunatic-fox) [reply icons/web3js/web3-original.svg:3]: Here are some logo suggestions based on: [web3/web3.js/assets/logo/web3js.svg](https://github.com/web3/web3.js/blob/8c643c2f51ec96b10993e4905c827a16e297f9a5/assets/logo/web3js.svg?plain=1#L4)

`original`


`plain`

File: devicon.json (hunk 2 above)
Comment (lunatic-fox) [reply devicon.json:10474]: I know it's not the main color of the logo, but on their site they use `#f16822` as an accent color and it's present on the styled "3" in the logo.

Comment (Snailedlt) [reply devicon.json:10474]: @lunatic-fox fair. I'll change the suggestion to that color

**Review by lunatic-fox — DISMISSED** [review]
Looks great now! 👍🏼 
Thank you for this contribution! 🚀
image

File: devicon.json [hunk 5]
```diff
@@ -10455,6 +10455,27 @@
...
+            "blockchain",
+            "ecommerce"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
```
Comment (lunatic-fox) [inline devicon.json:10473]: Whoops!
```suggestion
                "plain"
```

**Review by lunatic-fox — APPROVED** [review]

**Review by weh — APPROVED** [review]
LGTM :+1:

**Review by canaleal — APPROVED** [review]

**Pull request merged**